packages/react-storage/src/components/StorageManager/__tests__/StorageManager.test.tsx

```diff
@@ -286,4 +289,24 @@ describe('StorageManager', () => {
     expect(mockClearFiles).toHaveBeenCalledTimes(1);
     expect(mockClearFiles).toHaveBeenCalledWith();
   });
+
+  it('should throw an error if both prefix and accessLevel are specified', () => {
+    expect(() =>
+      render(
+        <StorageManager
+          prefix="prefix/"
+          accessLevel="private"
+          maxFileCount={1}
+        />
+      )
+    ).toThrow();
+    expect(errorSpy).toHaveBeenCalled();
+  });
+
+  it('should throw an error if both prefix and path are specified', () => {
```

Comment:
Can we add a test that accessLevel is private by default?

Reply:
Good point, working on it now